### PR TITLE
WebotsJS: Translate device

### PR DESCRIPTION
**Pull request by BenjaminDeleze**
Translate the device and the nodes that need a robot ancestor such that they always appear above the floor:

To review along: https://github.com/cyberbotics/webots-cloud/pull/107

Note: I will modify the `testing.webots.cloud` in `webots.cloud` before merging.

https://testing.webots.cloud/run?version=R2023b&url=https://github.com/cyberbotics/webots/blob/doc-add-robots/projects/robots/gctronic/elisa/protos/Elisa3DistanceSensor.proto&type=undefined

https://testing.webots.cloud/run?version=R2023b&url=https://github.com/cyberbotics/webots/blob/doc-add-robots/projects/robots/franka_emika/panda/protos/PandaHand.proto&type=undefined

### CODE REVIEW

File: resources/web/wwi/WebotsView.js [hunk 1]
```diff
@@ -408,6 +414,40 @@ export default class WebotsView extends HTMLElement {
...
+    let dbUrl = window.location.href;
+    if (!dbUrl.includes('webots.cloud'))
+      dbUrl = 'https://testing.webots.cloud/';
+    dbUrl = new URL(dbUrl).hostname;
+
+    if (protoUrl.includes('raw.githubusercontent.com')) {
+      protoUrl = protoUrl.replace('raw.githubusercontent.com', 'github.com');
+      protoUrl = protoUrl.split('/');
+      protoUrl.splice(5, 0, 'blob');
+      protoUrl = protoUrl.join('/');
```
Comment (ad-daniel) [inline resources/web/wwi/WebotsView.js:427]: some of these can probably be chained

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Concerning the webots.cloud PR, although the file change is fine, is it normal that it's showing 1500 commits? Is it because testing wasn't up to date or something?

**Comment by BenjaminDeleze** [comment]
> Concerning the webots.cloud PR, although the file change is fine, is it normal that it's showing 1500 commits? Is it because testing wasn't up to date or something?

Yes I think that we did not merge master in develop for a long time (and we have sometime sync problem with those branch)

File: resources/web/wwi/WebotsView.js (hunk 1 above)
Comment (BenjaminDeleze) [reply resources/web/wwi/WebotsView.js]: I could chained one, but as the splice directly apply on the string and does not return anything, I cannot chain further

**Review by ad-daniel — APPROVED** [review]

**Comment by BenjaminDeleze** [comment]
Can approve again pls? I just changed the url from `testing.webots.cloud` to `webots.cloud`

**Review by ad-daniel — APPROVED** [review]

**Pull request merged**